<launch>
	<include file="$(find ar_track_alvar)/launch/pr2_indiv_no_kinect.launch">
		<param name="cam_image_topic" value="/prosilica/image_rect" />
		<param name="cam_info_topic" value="/prosilica/camera_info" />
		<param name="output_frame" value="/head_cam_link" />
		<param name="marker_size" value="3.0" />
	</include>
</launch>
